test/utils/topic.test.ts

```diff
@@ -0,0 +1,26 @@
+import { buildContentTopic, isValidTopic } from '../../src/utils/topic'
+
+describe('topic utils', () => {
+  describe('isValidTopic', () => {
+    it('validates topics correctly', () => {
+      expect(isValidTopic(buildContentTopic('foo'))).toBe(true)
```

Comment:
Appreciate the robust tests. It might be helpful to just generate a bunch of random topics (the way we do when creating invites) and ensure that those all pass as well.

Reply:
good callout!

added random topic tests here: [test: add random topic testing](https://github.com/xmtp/xmtp-js/pull/495/commits/f3f94afbbc212683a941aae89630762ca3193fdf)